website/static/js/filepage/revisions.js

```diff
@@ -152,14 +205,17 @@ var FileRevisionsTable = {
         return self;
     },
     view: function(ctrl) {
-        return m('#revisionsPanel.panel.panel-default', [
+        var fileTags = ctrl.file.file_tags.join(',');
+        return m('', [
+            // Revisions Table
+            m('#revisionsPanel.panel.panel-default', [
```

Comment:
It seems to have renaming in this pr changing revision to detail, perhaps, change the id name as well.